### PR TITLE
[Editor] Don't add an already added editor (bug 1890367)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/integration/ink_editor_spec.mjs [hunk 1]
```diff
@@ -459,4 +459,45 @@ describe("Ink Editor", () => {
...
+
+    beforeAll(async () => {
+      pages = await loadAndWait("empty.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that a draw can be undone", async () => {
```
Comment (Snuffleupagus) [inline test/integration/ink_editor_spec.mjs:474]: This test name seems wrong here, too much copy-paste?

**Review by Snuffleupagus — APPROVED** [review]
r=me, with comment fixed.

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Looks good to me too; thank you for fixing this!

**Pull request merged**